Update allocation.md (#501)
kubecost/docs: allocation.md

cpuCoreUsageAverage | Average number of CPU cores used while running.
cpuCoreHours | Cumulative CPU core-hours allocated.
cpuCost | Cumulative cost of allocated CPU core-hours.
cpuCostAdjustment | Change in cost after allocated CPUs have been reconciled with updated node cost
cpuEfficiency | Ratio of `cpuCoreUsageAverage`-to-`cpuCoreRequestAverage`, meant to represent the fraction of requested resources that were used.
gpuCount | Number of GPUs allocated to the workload.
gpuHours | Cumulative GPU-hours allocated.
gpuCost | Cumulative cost of allocated GPU-hours.
gpuCostAdjustment | Change in cost after allocated GPUs have been reconciled with updated node cost

> networkTransferBytes | Cost of network data being sent

Reviewer: Not a cost field.

Reviewer: ```suggestion
networkTransferBytes | Total bytes sent from the workload
```

> networkTransferBytes | Cost of data being sent outside the network
> networkReceiveBytes | Cost of data being received by the network

Reviewer 2: ```suggestion
networkTransferBytes | Cost of network data being sent
networkReceiveBytes | Cost of network data being received
```

I'm pretty sure this is better wording, but please override if someone told you otherwise!

> networkReceiveBytes | Cost of network data being received

Reviewer: Not a cost field.

Reviewer: ```suggestion
networkReceiveBytes | Total bytes received by the workload
```

networkCost | Cumulative cost of network usage.
networkCostAdjustment | Matches network costs to reconciled assets and divides cost
loadBalancerCost | Cumulative cost of allocated load balancers.
loadBalancerCostAdjustment | Matches load balancer costs to reconciled assets and divides cost
pvBytes | Average number of bytes of PersistentVolumes allocated while running.
pvByteHours | Cumulative PersistentVolume byte-hours allocated.
pvCost | Cumulative cost of allocated PersistentVolume byte-hours.

> pvs | Allocation of PersistentVolumeClaim costs to specific pods

Reviewer 2: ```suggestion
pvs | Map of PersistentVolumeClaim costs that have been allocated to the workload
```

cpuCoreUsageAverage | Average number of CPU cores used while running.
cpuCoreHours | Cumulative CPU core-hours allocated.
cpuCost | Cumulative cost of allocated CPU core-hours.
cpuCostAdjustment | Change in cost after allocated CPUs have been reconciled with updated node cost
cpuEfficiency | Ratio of `cpuCoreUsageAverage`-to-`cpuCoreRequestAverage`, meant to represent the fraction of requested resources that were used.
gpuCount | Number of GPUs allocated to the workload.
gpuHours | Cumulative GPU-hours allocated.
gpuCost | Cumulative cost of allocated GPU-hours.
gpuCostAdjustment | Change in cost after allocated GPUs have been reconciled with updated node cost
networkTransferBytes | Cost of network data being sent
networkReceiveBytes | Cost of network data being received
networkCost | Cumulative cost of network usage.

> networkCostAdjustment | Matches network costs to reconciled assets and divides cost

Reviewer 3: Do we want to update this line too?

loadBalancerCost | Cumulative cost of allocated load balancers.

> loadBalancerCostAdjustment | Change in cost after allocated load balancers have been reconciled with updated node cost. Splits changes in cost evenly across allocations.

Reviewer 3: "updated load balancer cost"

pvBytes | Average number of bytes of PersistentVolumes allocated while running.
pvByteHours | Cumulative PersistentVolume byte-hours allocated.
pvCost | Cumulative cost of allocated PersistentVolume byte-hours.
pvs | Map of PersistentVolumeClaim costs that have been allocated to the workload

> pvCostAdjustment | Change in cost after allocated persistent volumes have been reconciled with updated node cost

Reviewer 3: "updated PV cost"
